test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -211,6 +211,7 @@ def run_data_service_test(ec2_connection, ec2_instance_ami, tensorflow_training,
     python_invoker = get_python_invoker(ec2_instance_ami)
     _, tensorflow_version = test_utils.get_framework_and_version_from_tag(tensorflow_training)
     ec2_connection.run(f"{python_invoker} -m pip install --upgrade pip")
+    ec2_connection.run(f"{python_invoker} -m pip install 'protobuf>=3.20,<3.21'")
```

Comment:
This was not required while releasing TF2.7 earlier. Please suggest if this change should also b applied in the dockerfile

Reply:
I believe this change should be in the dockerfile. This was not required previously as 3.20.1 was the highest released version (excluding release candidates) until May 25th. https://pypi.org/project/protobuf/#history

And our images seem incompatible at this time with the recently released version.

Reply:
Would it be possible to take on an exploratory task (low priority) to rework this test so that it runs TFContainer - TFContainer, rather than TFContainer - Host? It might be a long-term fix for a test that requires a server-client setup with both installing TF-dependencies.

Same for the TF MNIST inference test as well.